### ISSUE TITLE
Issue with dataloader

### ISSUE THREAD

**Issue by Anant4830** [juhongm999/hsnet#31]
Hello juhongm999,

Thank you for sharing the code. I am trying to implement hsnet but getting some error mentioned below. Kindly advise me on this.

I am also trying to train on pascal dataset. I am getting an error and not able to enter the for loop. Here is the code:-
for idx, batch in enumerate(dataloader):
# 1. Hypercorrelation Squeeze Networks forward pass
print("idx and batch are ========= ", idx,"\t", batch)
batch = utils.to_cuda(batch)
logit_mask = model(batch['query_img'], batch['support_imgs'].squeeze(1), batch['support_masks'].squeeze(1))
pred_mask = logit_mask.argmax(dim=1)

Here, I am not able to enter the for loop.

How did you solve it? any solution is appreciated.
